src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -1149,6 +1129,24 @@ private static File getBackupSaveLocation(final Date date)
         return new File(saveDir, String.format(FILENAME_MINECOLONIES_BACKUP, new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss").format(date)));
     }
 
+    public static void addToZipFile(final String fileName, final ZipOutputStream zos, final File folder)
+    {
+        final File file = new File(folder, fileName);
+        try (FileInputStream fis = new FileInputStream(file))
+        {
+            zos.putNextEntry(new ZipEntry(fileName));
+            Files.copy(file, zos);
+            fis.close();
+        }
+        catch (final Exception e)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)